ci/deps/azure-38-locale.yaml

```diff
@@ -34,7 +34,7 @@ dependencies:
   - xlsxwriter
   - xlwt
   - moto
-  - pyarrow>=0.15
+  - pyarrow=1.0.0
```

Comment:
we have other builds which get later version i think, right?

Reply:
do we have build which test a lower version as well? (and appropraitely skips the string dtype tests)

Reply:
we have #37306 for this decision. This was changed in https://github.com/pandas-dev/pandas/pull/35259/commits/1ed0585e79449ba524a5a64cd1abef23d212983c for ci testing. Can revert this b4 merge.

Reply:
> do we have build which test a lower version as well? (and appropraitely skips the string dtype tests)

see https://github.com/pandas-dev/pandas/pull/35259#issuecomment-712966527